agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -344,3 +356,345 @@ func TestTwoTasksSharedLocalVolume(t *testing.T) {
+	executionRole := os.Getenv("ECS_FTS_EXECUTION_ROLE")
+	if executionRole == "" {
+		t.Skip("ECS_FTS_EXECUTION_ROLE was not set to run this test")
+	}
+
+	// Setup SSM parameter for gMSA test
+	parameterName := "FunctionalTest-gMSA-SSM"
+	testCredSpecData := `{
+    "CmsPlugins":  [
+                       "ActiveDirectory"
+                   ],
+    "DomainJoinConfig":  {
+                             "Sid":  "S-1-5-21-975084816-3050680612-2826754290",
+                             "MachineAccountName":  "gmsa-acct-test",
+                             "Guid":  "92a07e28-bd9f-4bf3-b1f7-0894815a5257",
+                             "DnsTreeName":  "gmsa.test.com",
+                             "DnsName":  "gmsa.test.com",
+                             "NetBiosName":  "gmsa"
+                         },
+    "ActiveDirectoryConfig":  {
+                                  "GroupManagedServiceAccounts":  [
+                                                                      {
+                                                                          "Name":  "gmsa-acct-test",
+                                                                          "Scope":  "gmsa.test.com"
+                                                                      }
+                                                                  ]
+                              }
+}`
+	region := *ECS.Config.Region
+	ssmClient := ssm.New(session.New(), aws.NewConfig().WithRegion(region))
+	input := &ssm.PutParameterInput{
+		Description: aws.String("Resource created for the ECS Agent Functional Test: TestGMSASSMFile"),
+		Name:        aws.String(parameterName),
+		Value:       aws.String(testCredSpecData),
+		Type:        aws.String("String"),
+	}
+
+	// create parameter in parameter store if it does not exist
+	_, err := ssmClient.PutParameter(input)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case ssm.ErrCodeParameterAlreadyExists:
+				t.Logf("Parameter %v already exists in SSM Parameter Store", parameterName)
+				break
+			default:
+				require.NoError(t, err, "SSM PutParameter call failed")
+			}
+		}
+	}
+
+	// read parameter back to obtain ARN
+	paramInput := &ssm.GetParameterInput{
+		Name: aws.String(parameterName),
+	}
+	data, err := ssmClient.GetParameter(paramInput)
+	require.NoError(t, err)
+
+	// Obtain ARN of SSM param
+	gmSAParameterARN := *data.Parameter.ARN
+
+	// Setup agent
+	RequireDockerAPIVersion(t, ">=1.40")
+
+	agentOptions := AgentOptions{
+		ExtraEnvironment: map[string]string{
+			"ZZZ_SKIP_DOMAIN_JOIN_CHECK_NOT_SUPPORTED_IN_PRODUCTION": "true",
+			"ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION":                  "1m",
+		},
+	}
+
+	agent := RunAgent(t, &agentOptions)
+	defer agent.Cleanup()
+
+	agent.RequireVersion(">=1.33.0")
+
+	// Setup taskdef
+	tdOverrides := make(map[string]string)
+	tdOverrides["$$$TEST_REGION$$$"] = aws.StringValue(ECS.Config.Region)
```

Comment:
There is not TEST_REGION in your task definition, you don't need to pass this in.

Reply:
Removed.